src/games/strategy/triplea/ui/TripleaMenu.java

```diff
@@ -407,30 +407,21 @@ public void actionPerformed(final ActionEvent e) {
   private void addUnitNationDrawMenu(final JMenu parentMenu){
     final JMenu unitSizeMenu = new JMenu();
     unitSizeMenu.setMnemonic(KeyEvent.VK_N);
-    unitSizeMenu.setText("Show Unit Nation");
+    unitSizeMenu.setText("Set Flag Display Mode");
```

Comment:
The word "Set" is not needed in a menu item, it is already implied a bit. Can be shortened down to "Flag Display Mode"